tardis/gui.py

```diff
@@ -8,32 +41,61 @@
 from matplotlib.figure import *
 from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
 from matplotlib.backends.backend_qt4 import NavigationToolbar2QT as NavigationToolbar
-import os
-if os.environ.get('QT_API', None) is None:
+if os.environ.get('QT_API', None)=='pyqt':
     from PyQt4 import QtGui, QtCore
-else:
+elif os.environ.get('QT_API', None)=='pyside':
     from PySide import QtGui, QtCore
-    
+else:
+    raise ImportError('QT_API was not set! Please exit the IPython console'+
+        ' and at the bash prompt use : \n\n export QT_API=pyside \n or \n'+
```

Comment:
Are you sure about that? I remember reading that \ was not the preferred way of line breaking and one should use the implied line breaks inside (), [] and {} whenever possible and use \ only as a last resort. I am not an expert on PEP though. Just wanted to confirm before I changed these.